### PR TITLE
Bugfix/#5497 user cannot save comment at the address field on the order

### PR DESCRIPTION
**Pull request by NadiiaStoiko**
Before: User doesn't see the comment to the address in the order form if they enter the allowed 255 characters in the 'Comment to the address' field in step 2
image
After:User can save the comment to the address on the order form if they enter the allowed number of characters in the 'Comment to the address'.

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts [hunk 1]
```diff
@@ -174,6 +174,7 @@ export class UbsAdminOrderComponent implements OnInit, OnDestroy, AfterContentCh
   }
 
   initForm() {
+    console.log(' this.userInfo.recipientPhoneNumber', this.userInfo.recipientPhoneNumber);
```
Comment (katia9261) [inline src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts]: Remove comment

File: src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts [hunk 2]
```diff
@@ -283,6 +284,8 @@ export class UbsAdminOrderComponent implements OnInit, OnDestroy, AfterContentCh
   }
 
   getFormGroup(name: string): FormGroup {
+    // console.log('name',name);
```
Comment (katia9261) [inline src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts]: Here is the same )

**Review by katia9261 — APPROVED** [review]

**Review by KKatiaa — APPROVED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts (hunk 1 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts]: sorry, comments are removed

File: src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts (hunk 2 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts]: sorry, comments are removed

**Review by atsa21 — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Review by AlexW8719 — APPROVED** [review]

**Review by SokhaTaras — APPROVED** [review]

**Pull request merged**